extension.json

```diff
@@ -177,19 +177,29 @@
 		]
 	},
 	"Hooks": {
-		"ApiMain::moduleManager": "MediaWiki\\Extension\\Thanks\\Hooks::onApiMainModuleManager",
+		"ApiMain::moduleManager": "main",
+		"BeforePageDisplay": "main",
+		"DiffTools": "main",
+		"DifferenceEngineViewHeader": "main",
+		"GetAllBlockActions": "main",
+		"GetLogTypesOnUser": "main",
+		"HistoryTools": "main",
+		"LocalUserCreated": "main",
+		"LogEventsListLineEnding": "main",
+		"PageHistoryBeforeList": "main",
 		"BeforeCreateEchoEvent": "MediaWiki\\Extension\\Thanks\\Hooks::onBeforeCreateEchoEvent",
```

Comment:
You can use normal methods instead of static ones even though there's no interface